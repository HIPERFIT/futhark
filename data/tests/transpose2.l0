fun [[[int]]] main([[[int]]] a) =
  let b = transpose(1,1,a) in
  b
