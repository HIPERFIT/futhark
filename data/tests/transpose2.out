Result of evaluation: [[[1, 2, 3], [10, 20, 30], [100, 200, 300]], [[4, 5, 6], [40, 50, 60], [400,
                                                                        500,
                                                                        600]]]
